frontend/src/concepts/pipelines/content/tables/pipelineRun/CustomMetricsColumnsModal.tsx

```diff
@@ -44,54 +47,47 @@ export const CustomMetricsColumnsModal: React.FC<CustomMetricsColumnsModalProps>
 
   return (
     <Modal
-      bodyAriaLabel="Metrics column names"
       tabIndex={0}
-      hasNoBodyWrapper
       aria-label="Custom metrics columns modal"
-      variant={ModalVariant.small}
-      title="Customize metrics columns"
-      description={
-        <Stack hasGutter className="pf-v6-u-pb-md">
-          <StackItem className="pf-v6-u-mt-sm">
-            Select up to 10 metrics that will display as columns in the table. Drag and drop column
-            names to reorder them.
-          </StackItem>
+      variant="small"
+      isOpen
+      onClose={onClose}
+    >
+      <ModalHeader
+        title="Customize metrics columns"
+        description={
```

Comment:
Is there a reason why the header description is not in `ModalBody`?

Reply:
This is how PF designed the component https://www.patternfly.org/components/modal#modalheader